api/serializers/assets.py

```diff
@@ -437,6 +437,12 @@ class Meta:
         fields = ("asset_type", "model_number", "count_by_status")
 
 
+class DepartmentAssetSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = models.Asset
+        fields = ("id", "uuid", "asset_category")
```

Comment:
If we are to retain this and not use the `AssetSerializer` we should include the serial number and asset code (and probably the asset type). also exclude the id since we fetch assets by uuid.

Reply:
I have included those feilds